### ISSUE TITLE
Compile error ( go get)

### ISSUE THREAD

**Issue by arriqaaq** [manifoldco/promptui#48]
Hi,

I am getting the below error while compilation

"../../../../github.com/manifoldco/promptui/prompt.go:88:11: cannot use p.stdin (type io.Reader) as type io.ReadCloser in assignment:
	io.Reader does not implement io.ReadCloser (missing Close method)"

Please implement the method and update so that I can start using it

**Comment by nimajalali** [comment]
This is a duplicate of issue #46, waiting on PR #47 to land so it'll be resolved.

@arriqaaq not a long term fix but you can pull the branch referenced in #47 to unblock immediately, or just make the 2 line change yourself locally (thats what I did).

**Comment by luizbranco** [comment]
As @nimajalali mentioned, it should've been closed by https://github.com/manifoldco/promptui/pull/47
